examples/next/pages/_document.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Document, { Html, Head, Main, NextScript } from 'next/document'
+import { applyColorModeFromLocalStorage } from 'theme-ui'
+
+class MyDocument extends Document {
+  static async getInitialProps(ctx) {
+    const initialProps = await Document.getInitialProps(ctx)
+    return initialProps
+  }
+
+  render() {
+    return (
+      <Html>
+        <Head />
+        <body>
+          <script
```

Comment:
Thanks for looking into this! I wonder if it might make more sense from an API standpoint to make this a component, since this and `gatsby-ssr.js` both use a script element

Reply:
Yeah that's a good idea! I was thinking that after I submitted this PR. The reason I didn't do that is because in `gatsby-ssr.js` it's using a js function and not JSX. But I will update